subjects/java/checkpoints/top-frequents/README.md

```diff
@@ -0,0 +1,55 @@
+## Top Frequents
+
+### Instructions
+
+Given a non-empty array of integers, return the k most frequent elements. Use a hash map and a heap for efficient lookup and retrieval.
```

Comment:
I would add what is the sorting logic for the returned array: is it supposed to be ordered by "appearance" (smallest index in "original" array = smallest index in the returned array) or should it be based on "standard" ascending sorting of integer?

From the examples, both logics could be valid.

Reply:
it should be ordred by appearance first then if there is there is frequent equality the one that have the smallest index in the original array will be first.